docs/src/joint/api/connectionPoints/anchor.html

```diff
@@ -1,4 +1,33 @@
-<p>The <code>'anchor'</code> connection point function is the simplest connection point function. It places the connection point at the link end's anchor point (determined either by the <code>anchor</code> <a href="#anchors">function</a> or by the <code>defaultAnchor</code> <a href="#dia.Paper.prototype.options.defaultAnchor">paper option</a>).
+<p>The <code>'anchor'</code> connection point function is the simplest connection point function. It places the connection point at the link end's anchor point (determined either by the <code>anchor</code> <a href="#anchors">function</a> or by the <code>defaultAnchor</code> <a href="#dia.Paper.prototype.options.defaultAnchor">paper option</a>). It accepts several additional arguments, which can be passed within the <code>connectionPoint.args</code> property:</p>
+
+<table>
+    <tr>
+        <th>offset</th>
+        <td><i>number | object</i></td>
+        <td>An object with <code>x</code> and <code>y</code> properties. The connection point will be moved:
+            <ul>
+                <li>
+                    from the anchor point by <code>x</code> points in the direction of the first link path end segment.
+                </li>
+                <li>
+                    by <code>y</code> points in the direction of the first link path end segment rotated by 90 degrees anti-clockwise around the anchor point.
+                </li>
+            </ul>
+            If a number is provided, it will be used as the <code>x</code> offset.
+        </td>
+    </tr>
+    <tr>
+        <th>align</th>
+        <td><i>'top' | 'bottom' | 'left' | 'right'</i></td>
+        <td>Align the connection point with the first link path end segment opposite point.</td>
```

Comment:
<td>Offset the connection point to the point given by projecting the first vertex onto the vector which points from the anchor point in the direction specified. (If there are no vertices, use the projection of the other anchor point instead.) Notably, if the reference point is not the <q>direction</q>-most point of the two, the connection point is set to be the same as the anchor point. Let us illustrate that outcome and the other possible outcome on the `'top'` direction:
<ul>
    <li>The reference point lies below the anchor point. The anchor point is the topmost point of the two, so the connection point receives both the `x` and `y` coordinates from the anchor point. The connection point is set to be coincident with the anchor point.</li>
    <li>The reference point lies above the anchor point. In this case, the reference point is the topmost point of the two, so the connection point receives the `y` coordinate from the reference point. The connection point still receives the `x` coordinate from the anchor point. This means that the connection point is offset from the anchor point in the `'top'` direction. The three points create a right-angled triangle, with the right angle at the connection point.</li>
</ul>
</td>